pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -37,7 +37,12 @@ func InstanceSpecChecksum(spec servicecatalog.InstanceSpec) string {
 	specString += fmt.Sprintf("planName: %v\n", spec.PlanName)
 
 	if spec.Parameters != nil {
-		specString += fmt.Sprintf("parameters:\n\n%v\n\n", string(spec.Parameters.Raw))
+		params := spec.Parameters
+		if params.SecretRef != nil {
+			specString += fmt.Sprintf("parameters.secretRef: %v\n", params.SecretRef.Name)
```

Comment:
Am I correct that this means that if we're recalculating the checksum (for whatever reason) we won't detect a change in the secret's data?  Is this the normal behavior for kube?  Feels a little odd that we won't detect a change in the secret values yet if something else causes us to use the secert in a request we will grab the new values. Almost like we're 1/2 blind to the new data.  :-)